crates/net/discv4/src/proto.rs

```diff
@@ -261,7 +261,7 @@ impl EnrResponse {
     /// See also <https://github.com/ethereum/go-ethereum/blob/9244d5cd61f3ea5a7645fdf2a1a96d53421e412f/eth/protocols/eth/discovery.go#L36>
```

Comment:
mind moving this to `crates/ethereum-forks/src/forkid.rs` too alongside `EnrForkIdEntry`? makes sure spec changes like making the fork ID list not only a single value list, as mentioned in spec, has to be implemented once and is not missed anywhere.

Reply:
this is the discv4 type, this should remain in discv4